playbook/app/pb_kits/playbook/pb_lightbox/Carousel/index.tsx

```diff
@@ -20,8 +20,8 @@ type CarouselType = {
 export default function Carousel({
   currentIndex,
   photos,
-  onClick = ()=>{},
-  onChange = ()=>{},
+  onClick,
+  onChange,
```

Comment:
You can also do:

```tsx
onClick = () => undefined,
```

for these which would be more consistent as a void function.

Reply:
I'll update! Thanks!